extensions/amp-analytics/0.1/transport.js

```diff
@@ -21,7 +21,7 @@ import {timer} from '../../../src/timer';
 import {removeElement} from '../../../src/dom';
 
 /** @const {string} */
-const TAG_ = 'AmpAnalytics.Transport';
+const TAG_ = 'amp-analytics.Transport';
```

Comment:
It doesn't really matter. The important changes are those which would be used with `user.error` and similar. If it is the case here the best string would be just `amp-analytics` because users have no idea what this `Transport` thing is.